src/search/templates/search/search.html

```diff
@@ -86,7 +87,8 @@ <h3 class="panel-title">{% trans 'Latest posts' %}</h3>
                 <div class="row compose-content-div">
               	  <div class="compose-content-textarea-div col-xs-8">
 		              <label for="compose-content">{% trans 'Content' %}:</label><br/>
-		              <textarea class="form-control compose-content" rows="10" name="post" id="compose-content"></textarea>
+		              <textarea class="form-control compose-content" rows="10" name="post" id="compose-content" style="display: block"></textarea>
```

Comment:
Why are you `style="display: block"` the textarea?

Reply:
This is the required code, as written in ISSUE below.

https://github.com/s-tip/stip-common/pull/73#discussion_r517782927

Reply:
It was deleted.